projects/1.20/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,580 @@
+{
+	"key.toggleRingAttraction": "切换吸引戒指开/关",
+	"key.categories.gobber2": "戈伯",
+
+	"block.gobber2.gobber2_lucky_block": "戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_deepslate": "深层戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_deepslate": "深层戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_nether": "下界戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_nether": "下界戈伯幸运方块",
+	"block.gobber2.gobber2_lucky_block_end": "末地戈伯幸运方块",
+	"item.gobber2.gobber2_lucky_block_end": "末地戈伯幸运方块",
+
+	"block.gobber2.gobber2_ore": "戈伯矿石",
+	"item.gobber2.gobber2_ore": "戈伯矿石",
+	"block.gobber2.gobber2_ore_deepslate": "深层戈伯矿石",
+	"item.gobber2.gobber2_ore_deepslate": "深层戈伯矿石",
+	"item.gobber2.gobber2_ore.line1": "需要下界合金等级的镐子挖掘",
+
+	"item.gobber2.gobber2_glob": "戈伯簇",
+	"item.gobber2.gobber2_ingot": "戈伯锭",
+	"block.gobber2.gobber2_block": "戈伯块",
+	"item.gobber2.gobber2_block.line1": "方便存储戈伯锭",
+	"item.gobber2.gobber2_block.line2": "破坏时掉落完整的块",
```

```suggestion
	"item.gobber2.gobber2_block.line2": "破坏时掉落完整方块",
```

诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

## 已修正
- [1151330](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/4198/commits/1151330a1d72c1a9b505fb721e03df732360fd5b)